dtonator/src/main/java/com/bizo/dtonator/config/DtoConfig.java

```diff
@@ -64,7 +64,7 @@ public DtoConfig(final TypeOracle oracle, final RootConfig root, final String si
   }
 
   public List<DtoProperty> getAllProperties() {
-    return list(getInheritedProperties()).with(getProperties());
+    return list(getInheritedProperties()).with(getProperties()).unique();
```

Comment:
Do you still need this given that AFAIU the properties should not be duplicated in the first place anymore?

Reply:
It looks like you are correct. My original attempt still required the unique() call but this version does not. I tried taking out the unique() call and the equals()/ hashcode() and everything still works correctly.